packages/helm-charts/eksportisto/templates/statefulset.yaml

```diff
@@ -1,14 +1,28 @@
+apiVersion: v1
```

Comment:
Will the fact that the service is added here have an impact on targeting the prev deployment with suffix 8?

Reply:
I believe suffix 8 is an old version that is unlikely to be upgraded, and is only still up while we move to suffix 11. @yorhodes so you're aware

Reply:
What is this service used for?